src/Regridder.jl

```diff
@@ -633,5 +633,49 @@ function cgll2latlonz(field; DIR = "cgll2latlonz_dir", nlat = 360, nlon = 720, c
     return new_data, coords
 end
 
+"""
+    truncate_dataset(date0, timeStart, timeEnd, datafile)
+
+Truncates given data set to only regrid the dates that are used in the simulation
+"""
+function truncate_dataset(date0, timeStart, timeEnd, datafile, datapath)
```

Comment:
it would be good to use `time_start` and `time_end` instead of `timeStart` and `timeEnd` so we're consistent in using snake case throughout our code